packages/@adobe/spectrum-css-temp/components/page/index.css

```diff
@@ -11,3 +11,12 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
+
+.spectrum {
+  isolation: isolate;
+}
+
+/* We only want to apply isolation to our top most level provider. */
+.spectrum .spectrum {
```

Comment:
Can we apply the `isolation: isolate` in JS instead of in CSS with an override? Feels safer not to rely on the cascade...

Reply:
So initially i went that route, but it involved adding a prop to the provider wrapper and if anyone wanted to override it, it'd be harder, i'll do it here so you can see what i did though, i can back it out if we dislike it

Reply:
Can't you tell based on the context whether it's the top-level?

Reply:
Provider wrapper gets wrapped in context, so i can't do the check for no prev context there

Reply:
ok, merged it with styles